erpnext/stock/report/stock_ledger/stock_ledger.py

```diff
@@ -19,16 +19,23 @@
 
 def execute(filters=None):
 	is_reposting_item_valuation_in_progress()
+	if filters.get("include_uom") and filters.get("type") != "Default UOM":
+		frappe.throw(
+			_(
+				f"Cannot select filters {frappe.bold ('Include UOM')} and {frappe.bold('Select UOM')} together!"
```

Comment:
don't use f-strings in the translation function. Use the `format()` method instead.